Infinite rows with CSS Grid (#14)
cybertec-postgresql/today-i-learned-content: posts/TEMPLATE/index.md

---
date: 2030-12-31 # Auto updates on publish.
title: Infinite rows with CSS Grid
author: lorenz.henk@cybertec.at
tags: ["css", "grid", "grid-auto-rows"] # max. 10 tags; lowercase; dash-separated
description: "The grid-auto-rows can be used as an alternative to grid-template-rows" # max. 300 chars.
---

> [`CSS Grid`](https://caniuse.com/#feat=css-grid) is used to layout your website in columns and rows.

Reviewer: ```suggestion
[CSS Grids](https://caniuse.com/#feat=css-grid) can be used to layout your website in columns and rows.
```
